test/typescript/array-access/t.test.ts

```diff
@@ -46,6 +46,24 @@ describe('main', () => {
     expectTypeOf(t('arrayOfObjects.0', { returnObjects: true })).toEqualTypeOf<{ foo: 'bar' }>();
   });
 
+  it('should work with `count`', () => {
+    expectTypeOf(t('key', { count: 1 })).toEqualTypeOf<'item' | 'items'>();
+  });
```

Comment:
I don't think this test should go here. `key` values are just strings:
```
// ...
key_one: 'item';
key_other: 'items';
// ...
```

This scenario should be tested in `misc` or `custom-types` workspaces, if it is not already.